server/tweetSearch.js

```diff
@@ -8,44 +8,54 @@ module.exports = function(client) {
             basePath: "search",
             requestsRemaining: 0,
             resetTime: 0,
+            addData: function(tweets) {
+                this.since_id = tweets.statuses.reduce(function(since, currTweet) {
+                    return since > currTweet.id ? since : currTweet.id;
+                }, this.since_id);
+                tweetStore = tweetStore.concat(tweets.statuses.sort(function(statusA, statusB) {
+                    return statusA.id - statusB.id;
+                }));
+            }
         },
         "statuses/user_timeline": {
             since_id: 0,
             basePath: "statuses",
             requestsRemaining: 0,
             resetTime: 0,
+            addData: function(tweets) {
```

Comment:
No, the tweets object has a "statuses" field in one and not in the other. As also mentioned in my comment above, the code that they do share might also not be shared for long.